src/search/dbOperations.ts

```diff
@@ -267,13 +267,32 @@ export class DBOperations {
     return db;
   }
 
-  public static async getDocsByPath(db: Orama<any>, path: string): Promise<any | undefined> {
+  public static async getDocsByPath(db: Orama<any>, path: string) {
     if (!db) throw new Error("DB not initialized");
     if (!path) return;
     const result = await search(db, {
       term: path,
       properties: ["path"],
       exact: true,
+      includeVectors: true,
```

Comment:
Did you figure out how to make `exact` work? Is it still returning partial matches?

Reply:
No. I created a report and shared it with this issue: https://github.com/oramasearch/orama/issues/850